src/modules/operations.js

```diff
@@ -0,0 +1,66 @@
+/*eslint-disable*/
+export  class Operations {
+  static createTask() {
+    const inputData = document.getElementById('inputText').value;
+    const saveData = document.getElementById('formText');
+    const storage = this.getAllTasks().length;
+    const task = [{ description: inputData, completed: false, index: storage }];
+    this.saveTask(task);
+    this.showTask();
+    saveData.reset();
+  }
+  static showTask() {
+    const div = document.querySelector('#todo-container');
+    div.innerHTML = '';
+    this.getAllTasks().forEach((task) => {
+      let checked = '';
+      let style = '';
+      let styleDescription = '';
+      const styleText='style="display:none"';
+      if (task.completed) {
+        styleDescription = 'style="text-decoration: line-through"';
+        checked = 'checked';
+        style = 'class="line"';
+      }
+      div.innerHTML += `<li><input class="checkbox-class" type="checkbox" id="checkbox-${task.index}" ${checked} /> <h3  id="d${task.index}" ${style} ${styleDescription} >${task.description}</h3><input class="edit-class" type="text" value="${task.description}" id="edit-${task.index}" ${styleText}/><button type="button" id="editBttn-${task.index}" class="editButton"></button><button type="button" class="deleteButton" id="delete-${task.index}"></button></li>`;
+    });
+  }
+
+  
+  static removeTask(index) {
+    if (index > -1) {
+      const newData = this.getAllTasks();
+      newData.splice(index, 1);
+      for (let i = index; i < this.getAllTasks().length - 1; i += 1) {
+        newData[i].index = String(i, 10);
```

Comment:
- [ ] When a task is deleted the remaining tasks, downward to the last have their `index` values updated to strings. Kindly note that the `index` value should always be a number (integer), not a string. Kindly fix that

  ```
  newData[i].index = i;
  ```

  ![image](https://user-images.githubusercontent.com/36542986/228806832-a19670ef-92a5-4603-8a71-1ed477dad8e7.png)